src/crepe/system/RenderSystem.cpp

```diff
@@ -25,7 +29,7 @@ void RenderSystem::update_camera() {
 	if (cameras.size() == 0) throw std::runtime_error("No cameras in current scene");
 
 	for (Camera & cam : cameras) {
-		this->context.camera(cam);
+		this->context.set_camera(cam);
```

Comment:
het lijkt erop of de set_camera nu altijd de laaste camera van de vector is. moet er niet gekeken worden welke camera actief is?

Reply:
Dat is een goede helemaal vergeten.